client/src/pages/EditProject.js

```diff
@@ -0,0 +1,327 @@
+import React, { Component } from 'react';
+import { Typography, withStyles, Button, TextField } from '@material-ui/core';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import validator from 'validator';
+import moment from 'moment'
+
+import { OutlinedSelect } from '../components/Inputs';
+import UploadImages from '../components/UploadImages';
+import FormValidator from '../helpers/form-validation';
+import { editProject } from '../api/projects';
+import { locations, industries } from '../dummyData/dropDownItems'
+import { withPageContext } from '../components/pageContext';
+
+const styles = {
+  pageContent: {
+    display: 'flex',
+    justifyContent: 'left'
+  },
+  projectPreviewContainer: {
+    display: 'flex',
+    width: '25%',
+    flexDirection: 'column',
+    marginRight: '20px',
+    marginLeft: '10px',
+    padding: '30px 50px 20px 30px',
+    boxShadow: '2px 0px 4px 2px #D3D3D3',
```

Comment:
I will look at the design when I pull down, but I would try to avoid box shadows unless it is a card like the project details card